spec/spec_helper.rb

```diff
@@ -65,6 +65,6 @@
 
   config.ignore_request do |request|
     # Ignore capybara requests within feature tests
-    request.uri =~ /__identify__/
+    request.uri =~ /(__identify__|example\.com)/
```

Comment:
No, because VCR hooks before webmock.  Ah, I'd forgotten you can turn it off...I'll remove and reimplement as an around block.